source/documentation/06-api-reference.md

```diff
@@ -134,6 +136,34 @@ These error codes provide more information about why a request failed.
 | General | P0920 | Request blocked by security rules | Our firewall blocked your request. See Troubleshooting section for details. |
 | General | P0999 | GOV.UK Pay is unavailable | The GOV.UK Pay service is temporarily down. |
 
+### API errors caused by payment statuses
+
+The API error codes in this section are driven by payment status.
+
+| Error code | Meaning | Cause |
+|:---|:---|:---|
+| P0050 | Payment provider returned an error | Multiple possible causes, for example a configuration problem with the payment provider, or incorrect login credentials; contact us, quoting the error code |
```

Comment:
Is there any particular reason the error codes aren’t listed in numerical order?

Reply:
No reason, have changed